src/main/java/com/r307/arbitrader/service/NotificationServiceImpl.java

```diff
@@ -98,7 +98,8 @@ public void sendEntryTradeNotification(Spread spread, BigDecimal exitTarget, Big
      */
     @Override
     public void sendExitTradeNotification(Spread spread, BigDecimal longVolume, BigDecimal longLimitPrice, BigDecimal shortVolume,
-                                          BigDecimal shortLimitPrice, BigDecimal entryBalance, BigDecimal updatedBalance, BigDecimal exitTarget,
+                                          BigDecimal shortLimitPrice, BigDecimal entryBalance, BigDecimal updatedBalance,
+                                          BigDecimal combinedBalance, BigDecimal exitTarget,
                                           boolean isForceExitPosition, boolean isActivePositionExpired) {
 
         final String exitSpreadString = String.format("Exit spread: %s\nExit spread target %s\n", spread.getOut(), exitTarget);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.